src/pages/db/index.astro

```diff
@@ -32,8 +33,9 @@ const description = "The SQL database platform for content-driven websites."
 	</div>
 </MainLayout>
 
-<style>
+<style is:global>
 	.db {
 		background-color: #060913 !important;
+		scroll-padding-top: 7.5rem;
 	}
 </style>
```

Comment:
This being global also scares me; I'd prefer these styles to be written as inline TW classes wherever they're needed.

In case you need a refresher: `!bg-[#060913] scroll-pt-28`